05/ContactForm.js

```diff
@@ -1,10 +1,113 @@
-import React from 'react';
+import React, { useReducer } from 'react';
 
-import account from './account';
+import './styles/ContactForm.css'
+
+// import account from './account';
+
+const initialState = {
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    subject: '',
+    message: '',  
+}
+const reducer = (state, action) => {
+    switch(action.type) {
+    case 'set-firstName': return {...state, firstName: action.payload}
+    case 'set-lastName': return {...state, lastName: action.payload}
+    case 'set-email': return {...state, email: action.payload}
+    case 'set-phone': return {...state, phone: action.payload}
+    case 'set-subject': return {...state, subject: action.payload}
+    case 'set-message': return {...state, message: action.payload}
```

Comment:
Nie dałoby się tego napisać bardziej "dynamicznie". 
Jeśli pól w formularzu byłoby 100, to co wtedy? :>